package.json

```diff
@@ -6,12 +6,14 @@
     "@testing-library/jest-dom": "^5.11.0",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.0.11",
+    "node-sass": "^5.0.0",
     "normalize.css": "^8.0.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.1"
+    "react-scripts": "^4.0.1",
+    "styled-components": "^5.2.3"
```

Comment:
It doesn't look like you use `styled-components`. So you could remove the dependency as well